pkg/network/master/egressip.go

```diff
@@ -177,18 +191,19 @@ func (eim *egressIPManager) check(retrying bool) (bool, error) {
 			continue
 		}
 
+		klog.Warningf("The node name is %s", node.name)
 		nn, err := eim.nodeInformer.Lister().Get(node.name)
 		if err != nil {
 			klog.Warningf("Couldn't get node %s ", node.ip)
```

Comment:
you need to return/break/continue/something here; as the code is now, it's going to try to dereference `nn` anyway.
also you should be including the error in the logged message